lib/roo/excelx/shared_strings.rb

```diff
@@ -18,7 +18,7 @@ def extract_shared_strings
       if doc_exists?
         # read the shared strings xml document
         doc.xpath("/sst/si").map do |si|
-          shared_string = ''
+          shared_string = +''
```

Comment:
What does the + change?

Reply:
It's shorthand for ''.dup since this file now has frozen string literal enabled.